core/src/main/java/com/mapzen/android/graphics/MapzenMap.java

```diff
@@ -649,7 +643,8 @@ public void onMarkerPick(final MarkerPickResult markerPickResult, final float po
         mapView.post(new Runnable() {
           @Override public void run() {
             if (markerPickResult != null) {
-              listener.onMarkerPick(new BitmapMarker(markerManager, markerPickResult.getMarker()));
+              listener.onMarkerPick(new BitmapMarker(markerManager, markerPickResult.getMarker(),
```

Comment:
This should be updated to use code to map the Tangram `Marker` to the existing `BitmapMarker`. Created a ticket to do this in future work: https://github.com/mapzen/android/issues/429